City-of-Helsinki/outdoors-sports-map
# Add default observation

- adds default observation for ice swimming and outdoor services

## Description

Add default observations:
 - adds default observations for ice swimming and outdoor services
 - changes the icons use to green and conditions to good

## Context

[HULK-1](https://andersinno.atlassian.net/browse/HULK-1)

## How Has This Been Tested?

- select `Ice swimming` sport  and outdoor services ( one or multiple)
- check if icons used on the map are green for all
- details of a service point should also show `Condition` in green
  
## Manual Testing Instructions for Reviewers

- select `Ice swimming` sport  and outdoor services ( one or multiple)
- check if icons used on the map are green for all
- details of a service point should also show `Condition` in green

## Screenshots

outdoor services icon and condition:
![outdoor_services_icons_and_conditions](https://user-images.githubusercontent.com/117433931/229501777-3d4754dc-ec42-4dcb-ac21-108915febfb0.png)

ice swimming icon and condition:
![ice_swimming_area_icon_and_condition](https://user-images.githubusercontent.com/117433931/229501829-9d50ad0e-5a9b-473a-b4eb-57481cc17d85.png)
